lib/util/clone.ts

```diff
@@ -1,11 +1,9 @@
-import { quickStringify } from './stringify';
+import { klona } from 'klona/json';
 
 /**
  * Creates a deep clone of an object.
- * @deprecated Use {@link structuredClone} instead.
  * @param input The object to clone.
  */
-export function clone<T>(input: T | null = null): T {
-  const stringifiedInput = quickStringify(input);
-  return stringifiedInput ? JSON.parse(stringifiedInput) : null;
+export function clone<T = unknown>(input: T): T {
+  return klona(input);
```

Comment:
Maybe make this change first? Leave structuredClone until a second?

Reply:
No problem